vllm/transformers_utils/config.py

```diff
@@ -758,3 +771,38 @@ def get_cross_encoder_activation_function(config: PretrainedConfig):
         return resolve_obj_by_qualname(function_name)()
     else:
         return nn.Sigmoid() if config.num_labels == 1 else nn.Identity()
+
+
+@cache
+def model_overwrite(model: str):
```

Comment:
```suggestion
def maybe_model_redirect(model: str):
```
I prefer to use `maybe_model_redirect` here.

Reply:
maybe_model_redirect  fine